bauhaus/main.py

```diff
@@ -5,6 +5,15 @@
 from bauhaus.workflows import availableWorkflows
 from bauhaus.utils import mkdirp
 
+def doWorkflowHelp(args):
+    if not args.workflow :
+        print "Please pick a workflow with the --workflow option:"
+        print ' '.join(availableWorkflows.keys())
+        return
+
+    wfg = availableWorkflows[args.workflow]()
+    wfg.help()
```

Comment:
I like this, but we need to actually implement this help() method everywhere now...